examples/kernel_ridge/plot_krc_rbf_parameters.py

```diff
@@ -0,0 +1,173 @@
+'''
+===============================================
+RBF Kernel Ridge Classifier parameters
+==================================================
+
+This example illustrates the effect of the parameters ``gamma`` and ``C`` of
+the Radial Basis Function (RBF) Kernel Ridge Classifier.
+
+Intuitively, the ``gamma`` parameter defines how far the influence of a single
+training example reaches, with low values meaning 'far' and high values meaning
+'close'. The ``gamma`` parameters can be seen as the inverse of the radius of
+influence of samples selected by the model as support vectors.
+
+The ``C`` parameter trades off misclassification of training examples against
+simplicity of the decision surface. A low ``C`` makes the decision surface
+smooth, while a high ``C`` aims at classifying all training examples correctly
+by giving the model freedom to select more samples as support vectors.
```

Comment:
As our Ridge models are parametrized in terms of alpha instead of C this paragraph will need to be rewritten are `alpha` is inversely proportional to `C`.